test/unit/os/env_spec.lua

```diff
@@ -305,4 +309,19 @@ describe('env.c', function()
       eq(output[4], 0)
     end)
   end)
+
+  describe('init_homedir', function()
+    itp('homedir set to $HOME if $HOME exists', function()
+      os_setenv('HOME', '/home/username', 1)
+      cimp.init_homedir()
+      eq(get_homedir(), '/home/username')
+    end)
+
+    itp('able to get an non-empty homedir with empty $HOME', function()
+      os_unsetenv('HOME')
+      cimp.init_homedir()
+      neq(get_homedir(), '')
+      neq(get_homedir(), nil)
```

Comment:
would be better to check that length is `> 4` or something like that.

    assert(#get_homedir() > 4)

Reply:
Done

> would be better to check that length is `> 4` or something like that.
> 
> ```
> assert(#get_homedir() > 4)
> ```

Done

Reply:
Done